addon/Utils.jsm

```diff
@@ -0,0 +1,68 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's find a better name for this. These functions are not being used outside of the simulation. So just call this file "Simulator.jsm"

Reply:
Done